Source/WebCore/rendering/RenderTable.cpp

```diff
@@ -1571,7 +1571,7 @@ std::optional<LayoutUnit> RenderTable::firstLineBaseline() const
     // doesn't define the baseline of a 'table' only an 'inline-table').
     // This is also needed to properly determine the baseline of a cell if it has a table child.
 
-    if ((isWritingModeRoot() && !isFlexItem()) || shouldApplyLayoutContainment())
+    if ((isWritingModeRoot() && !isFlexItem()) || shouldApplyLayoutContainment() || style().containerType() != ContainerType::Normal)
```

Comment:
This fixes 8 FAIL (regression) for `table`, but `table-cell` is still failing and I haven't found yet how. 
https://wpt.fyi/results/css/css-conditional/container-queries/nested-query-containers.html?label=experimental&label=master&aligned

Reply:
https://wpt.live/css/css-conditional/container-queries/nested-query-containers.html
![image](https://github.com/user-attachments/assets/c4ce43ee-2fba-466f-877c-31a17e70b166)

Reply:
ha RenderTableCell.cpp inherits from RenderBlock.cpp OK
Let's try.

Reply:
@nt1m that worked! I went one level up.